akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -581,10 +581,7 @@ object Flow {
    *
    * '''Cancels when''' downstream cancels
    */
-  @Deprecated
-  @deprecated(
-    "Use lazyInitAsync instead. (lazyInitAsync returns a flow with a more useful materialized value.)",
-    "2.5.12")
+  @deprecated("Use 'Flow.lazyFutureFlow' instead", "2.6.0")
   def lazyInit[I, O, M](flowFactory: I => Future[Flow[I, O, M]], fallback: () => M): Flow[I, O, M] =
```

Comment:
you said that my table was wrong for this one

> Flow.lazyInit becomes lazyFlow

but I think it's lazyFutureFlow

Reply:
You are right, nvm